src/clients/near.ts

```diff
@@ -130,7 +130,7 @@ interface BalanceResults {
   storage: StorageDeposit
 }
 
-type UtxoChain = ChainKind.Btc | ChainKind.Zcash
+export type UtxoChain = ChainKind.Btc | ChainKind.Zcash
```

Comment:
Perhaps this is a better fit for src/utils/chain.ts like `EVMChainKind`?
Alternatively, both can go to src/types/chain.ts